src/ReactAI.ts

```diff
@@ -1,32 +1,27 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import { PropertiesPluginIdentifier } from "@microsoft/applicationinsights-common";
 import { IPlugin } from "@microsoft/applicationinsights-core-js";
-import {
-  IAppInsightsCore,
-  IApplicationInsights,
-  IConfig,
-  IConfiguration,
-  IPageViewTelemetry,
-  ITelemetryItem,
-  ITelemetryPlugin
-} from "@microsoft/applicationinsights-web";
+import { ApplicationInsights as AppInsightsPlugin, IAppInsightsCore, IConfig, IConfiguration, IPageViewTelemetry, ITelemetryItem, ITelemetryPlugin, PropertiesPlugin } from "@microsoft/applicationinsights-web";
 import { Action, History, Location } from "history";
 import IReactAISettings from "./IReactAISettings";
 
+
 /**
  * Module to include Microsoft Application Insights in React applications.
  *
  * @export
- * @class ReactAI
+ * @class ReactAI  
  */
 class ReactAI implements ITelemetryPlugin {
   public extensionId: string = "ApplicationInsightsReactUsage";
   public ApplicationInsightsAnalyticsIdentifier: string = "ApplicationInsightsAnalytics";
   public processTelemetry: (env: ITelemetryItem) => void;
   public identifier = this.extensionId;
   public priority: number = 190;
-  public appInsights!: IApplicationInsights;
+  public appInsights!: AppInsightsPlugin;
+  private propertiesPlugin!: PropertiesPlugin;
```

Comment:
I apologize my previous comment was about `propertiesPlugin` being private. Shouldn't it be public? If it's private, what is using it, and why does it need to be there in the first place?

Reply:
You need it to access the setAuthenticatedUserContext() api

Reply:
I dont have the code change here that you had for accessing that api which was the original issue.